src/rules.js

```diff
@@ -1869,6 +1869,27 @@ module.exports = [
     },
     /**
     ==================================================================
+    OBJECTS
```

Comment:
I gave this a more generic name, objects, since there are other properties like [object-position](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) which may be covered later on

Reply:
I think this is fine, we try to follow Emmet syntax but Emmet doesn't seem to support these yet